pkgs/by-name/st/st-snazzy/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fontconfig,
+  harfbuzz,
+  libX11,
+  libXft,
+  ncurses,
+  gd,
+  glib,
+  pkg-config,
+}:
+
+stdenv.mkDerivation {
+  pname = "st-snazzy";
+  version = "0.8.5";
```

Comment:
How does that version correlate with the commit below?

Reply:
I took it from the default.nix in the original repo.
https://github.com/siduck/st/blob/main/default.nix
There's no other mention of version anywhere else in the project.
Do you suggest something like `version = "2024-09-06";` (date of the last commit)?

Reply:
asked on Matrix and people suggested `version = "0.8.5-unstable-2024-09-06";`, so I made the change.